actionpack/lib/action_dispatch/system_testing/browser.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+module ActionDispatch
+  module SystemTesting
+    class Browser # :nodoc:
+      attr_reader :name
```

Comment:
Can you add a new line under `attr_reader`?

Reply:
Done in f52e17f